### PR TITLE
[bug 1248040] Update GTM data attributes on contribute pages navigati…

### PR DESCRIPTION
**Pull request by JamesDLorence**
…on and buttons

Bug 1248040 - https://bugzilla.mozilla.org/show_bug.cgi?id=1248040

### CODE REVIEW

File: media/js/mozorg/contribute/contribute-2015-ga.js [hunk 1]
```diff
@@ -47,50 +31,6 @@ $(function () {
...
-            'data-element-action': 'social button click',
-            'data-tracking-flag': 'mozillians'
-        });
-    });
-
-    // Track other actions on landing page
-    $('.landing-notready .other-actions a').attr('data-element-location', 'not ready');
-
     // Track other actions on confirmation page
     $('#thankyou .other-actions a').each(function() {
```
Comment (alexgibson) [inline media/js/mozorg/contribute/contribute-2015-ga.js]: Looks like there is still some cleanup we could do by using data attributes on the confirmation/thank you page?

Comment (JamesDLorence) [inline media/js/mozorg/contribute/contribute-2015-ga.js]: Are you recommending I remove the other data attributes being added by JS?

Comment (alexgibson) [inline media/js/mozorg/contribute/contribute-2015-ga.js]: Yes, that's what I'm suggesting. Ideally this kind of thing could bring the template directly.

**Comment by JamesDLorence** [comment]
I just removed those last remaining data attribute additions form the JS as it looked like they weren't being used anymore.

**Comment by alexgibson** [comment]
r+ thanks @JamesDLorence

**Comment by JamesDLorence** [comment]
@alexgibson - This commit wrapped up the last planned commits for this data attribute work. When do you think this commit will go live in production?

**Comment by alexgibson** [comment]
It should be live shortly. I'll post in the bug and mark as resolved once it hits prod. Thanks @JamesDLorence!

**Pull request merged**